include/rf.h

```diff
@@ -49,6 +49,10 @@ namespace Rf {
 
 const std::vector<std::string> VENDORS { "3GPP", "" };
 
+// This will need updating when the version of Spec TS32.299 that we support
```

Comment:
Can you flesh out this comment a bit more (e.g. what is the expected format, why we've not included the extensions, where in the specs this is defined)?